atom/browser/web_contents_preferences.cc

```diff
@@ -155,8 +156,19 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
     command_line->AppendSwitchASCII(::switches::kEnableBlinkFeatures,
                                     blink_features);
 
-  // The initial visibility state.
   NativeWindow* window = NativeWindow::FromWebContents(web_contents);
+
+  // Inherit initial visibilty state from parent window in webviews
+  if (guest_instance_id && !window) {
```

Comment:
The `guest_instance_id` needs to be initialized above, otherwise it could be filled with random data.